larray/util/misc.py

```diff
@@ -131,20 +133,27 @@ def get_min_width(table, index):
     return max(longest_word(row[index]) for row in table)
 
 
-def table2str(table, missing, fullinfo=False, summarize=True, maxwidth=80, numedges='auto', sep='  ', cont='...',
-              keepcols=0):
+def table2str(table, missing, summarize=True, maxwidth=None, numedges='auto', sep='  ', cont='...',
+              keepcols=0, precision=None):
     """
     table is a list of lists
     :type table: list of list
     """
     if not table:
         return ''
+
+    from larray.util.options import OPTIONS, DISPLAY_WIDTH, DISPLAY_PRECISION
```

Comment:
I would much prefer if this (taking the default value into account) was handled in `LArray.__str__` instead